src/partials/footer-scripts.hbs

```diff
@@ -18,7 +18,7 @@
 <script defer src="{{{uiRootPath}}}/js/vendor/fontawesome.js"></script>
 <script async src="{{{uiRootPath}}}/js/vendor/highlight.js"></script>
 {{#with (resolvePage page.relativeSrcPath model=false)}}
-    {{#unless (eq asciidoc.attributes.stem undefined)}}
+    {{#unless (and (eq asciidoc.attributes.stem undefined) (eq page.attributes.stem undefined))}}
```

Comment:
The check `{{#unless (eq ./asciidoc.attributes.stem undefined)}}` should suffice. This checks whether `stem` is set on the AsciiDoc file (which can be inherited from the default setting in the playbook). There's no need to involve the `page-stem` attribute here.
```

```

Reply:
@mojavelinux

Yes, tried it this morning. Works fine as it is. Not sure what I was getting wrong last week. Ll close this PR as it's not needed.